pyproject.toml

```diff
@@ -10,4 +10,102 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "cheroot"
+description = "Highly-optimized, pure-python HTTP server"
+authors = [
```

Comment:
Should we start having an additional `maintainers` entry at some point?

Reply:
Yes, but not as part of this change.

Reply:
Fair.